.github/workflows/windows_build.yml

```diff
@@ -22,16 +20,18 @@ jobs:
     - name: Build
       shell: cmd
       run: |
-        choco sources add -n=roswin -s https://aka.ms/ros/public --priority 1
-        choco install ros-%ROS_DISTRO%-desktop_full -y --no-progress
+        set ChocolateyInstall=C:\opt\chocolatey
+        choco sources add -n=ros-win -s="https://aka.ms/ros/public" --priority=1
 
         call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat" -arch=amd64 -host_arch=amd64
-        call "C:\opt\ros\%ROS_DISTRO%\x64\setup.bat"
 
+        python -m pip install vcstool -q
+        python -m pip install colcon-common-extensions -q
         choco install console_bridge -y --no-progress
-        choco install clapack -y --no-progress
-        choco install openblas -y --no-progress
+        choco install boost -y --no-progress
+        choco install eigen -y --no-progress
+        refreshenv
```

Comment:
This needs to be switched to `call RefreshEnv.cmd` calling `refreshenv` automatically exits the bash and never builds or runs the test.

Reply:
Good find. Now that you mention it, I do vaguely recall trying that in a prior attempt and seeing the same thing. I just opened #91 to address. I'll check that it actually builds before merging